staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go

```diff
@@ -142,6 +142,7 @@ func (r *proxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	newReq := req.WithContext(context.Background())
 	newReq.Header = utilnet.CloneHeader(req.Header)
 	newReq.URL = location
+        newReq.Host = location.Host
```

Comment:
Are you able to add a  unit test to verify the change of this request? There is a `targetHTTPHandler` in test, so maybe you can reuse it as the mock.

Reply:
@resouer thanks for your suggestion，i've added unit test